go/mysql/collations/tools/colldump/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:22.04
```

Comment:
We're using Debian bases in most (if not all?) places, so let's use that here too.